tensorboard/webapp/widgets/range_input/range_input_component.ts

```diff
@@ -345,14 +345,21 @@ export class RangeInputComponent implements OnInit, OnDestroy {
     // range  [100, 500]  → delete upper value → single [100, X]
     // single [100, X]    → delete lower value → single [50, X]
     // range  [100, 500]  → delete lower value → range  [50, 500]
+    // single [100, X]    → add    upper value → range  [100, 500]
     if (this.isUpperValueRemoved(input, position)) {
+      this.upperValue = null;
```

Comment:
I don't think we should set this, if it can be avoided. It's an Input parameter - we should rely on ngrx processing the action and the selectors emiting updated values.